website/docusaurus.config.ts

```diff
@@ -4,13 +4,14 @@ import type * as Preset from "@docusaurus/preset-classic";
 import type { Config } from "@docusaurus/types";
 import type * as Plugin from "@docusaurus/types/src/plugin";
 import type * as OpenApiPlugin from "docusaurus-plugin-openapi-docs";
+import "dotenv/config";
```

Comment:
why do we need this?

Reply:
to support env variables in docusaurus.config.ts file